salesagility/SuiteCRM
# Add ability to have Notes on a Campaign

A Campaign may represent many different "things". For example an Email Campaign, a Trade Show, a Conference, etc.

Some times it is helpful to have "Notes" related to a "Campaign".

## How To Test This


Create a new Note, relate it to Type "Campaign" and choose an existing Campaign.
Open up the Campaign, look at History subpanel and see the new Note.

## Types of changes


### Final checklist
